pythonProject/theory questions/exam.txt

```diff
@@ -0,0 +1,72 @@
+Questions:
+2-marks:
+1) What is Selenium?
+selenium is a web application automation framework.
+it has more tools and libraries to automate application
+
+2) Command to install selenium.
+pip install selenium
+3) What are the locators in Selenium?
+ID,name,classname,xpath,linked text,partially linked text,css selector
+4) What is the difference between find_element_by_* and find_elements_by_* in Selenium?
```

Comment:
Why are we using find_element_by two times, where in the question it has been asked find_element and find_elements.? - 1mark.